### PR TITLE
Fix Preferences Speed Tab

### PR DESCRIPTION
**Pull request by Secretmapper**
- Pass some values (i.e. alt-speed-time-begin) as numerical values
- Make SelectRow use actual value
- Add Proper times to scheduled times

### CODE REVIEW

File: src/components/dialogs/PreferencesDialog/SpeedTabPanel/index.js [hunk 1]
```diff
@@ -1,23 +1,32 @@
 import React, { Component} from 'react';
 import { inject, observer } from 'mobx-react';
-import lodash from 'lodash';
 
 import TextRow from '../fields/TextRow';
 import CheckRow from '../fields/CheckRow';
 import CheckValueRow from '../fields/CheckValueRow';
 import SelectRow from '../fields/SelectRow';
 
+const generateTimes = () => {
```
Comment (edulan) [inline src/components/dialogs/PreferencesDialog/SpeedTabPanel/index.js]: Maybe it's interesting to extract so some utility module and add some tests to it. What do you think @Secretmapper ?

File: src/components/dialogs/PreferencesDialog/index.js [hunk 2]
```diff
@@ -78,4 +78,13 @@ class PreferencesDialog extends Component {
   }
 }
 
+function parseAsNumberIfNumber (str) {
```
Comment (edulan) [inline src/components/dialogs/PreferencesDialog/index.js]: This is also a canditate for extraction. Consider using lodash `isInteger` and `isNumber` methods:

https://lodash.com/docs/4.17.4#isInteger

Also conversion methods:

https://lodash.com/docs/4.17.4#toInteger

Beware of returning `undefined`.

Comment (Secretmapper) [reply src/components/dialogs/PreferencesDialog/index.js]: Huh, I just realized the commit I'm replacing (preferences dialog) removes the only reference to lodash (the times for SpeedTabPanel was generated using lodash).

I think it'd be best to keep deps at the minimum, I'll remove the lodash dependency and I'll extract this helper function to util

File: src/components/dialogs/PreferencesDialog/SpeedTabPanel/__fixtures__/schedule.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  '1': 'Sunday',
+  '2': 'Monday',
+  '4': 'Tuesday',
+  '8': 'Wednesday',
+  '16': 'Thursday',
+  '32': 'Friday',
+  '64': 'Saturday',
+}
+
+export const times = {
```
Comment (fcsonline) [inline src/components/dialogs/PreferencesDialog/SpeedTabPanel/__fixtures__/schedule.js]: Could we generate that?

**Comment by fcsonline** [comment]
If you can solve the `package.json` conflict, we can merge this. :muscle:

**Comment by Secretmapper** [comment]
Done :)

**Comment by fcsonline** [comment]
Rebase from master and simplify commit, please

**Comment by fcsonline** [comment]
I just cherrypicked your last commit with some fixes

**Pull request closed without merging**